tests/test_app_config.py

```diff
@@ -25,8 +25,8 @@ def test_app_testing_config(self):
         self.assertTrue(application.config["TESTING"])
         self.assertFalse(application.config["SQLALCHEMY_TRACK_MODIFICATIONS"])
         self.assertTrue(application.config["MOCK_EMAIL"])
-        self.assertEqual("postgresql:///bit_schema_test", application.config["SQLALCHEMY_DATABASE_URI"]
-        )
+        # self.assertEqual("postgresql:///bit_schema_test", application.config["SQLALCHEMY_DATABASE_URI"]
```

Comment:
Delete or uncomment

Reply:
mmm. The same idea as it was done in mentorship system, I'm thinking of allowing people to use their own remote db. Shouldn't this be the same case as BIT?

Reply:
I've just removed this line now. 👍

Reply:
uncommented 👍